test/rulesets/Base/mapreduce.jl

```diff
@@ -1,3 +1,30 @@
+"For testing this config re-dispatches Xrule_via_ad to Xrule without config argument"
```

Comment:
Why? We want to preserve the config, right? And let it hit the fallback in CRC?

Reply:
fair, yes, that would be better